### PR TITLE
applied new form-container to parts of constructor

### PR DESCRIPTION
**Pull request by Pipuss**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** original screenshot ** | ** updated screenshot ** |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/components/forms/material-form/material-form.js [hunk 1]
```diff
@@ -237,8 +237,8 @@ function MaterialForm({ material, id }) {
                 label='Застосування'
               >
                 {Object.values(purposeEnum).map((value) => (
-                  <option key={value} value={value}>
-                    {value}
+                  <option key={value.en} value={value.en}>
```
Comment (maarekua) [inline src/components/forms/material-form/material-form.js:240]: Unreadable, what are value.en and value.ua?

File: src/containers/constructor-features-container/constructor-features-container.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prop-types */
+// TODO: Implement propTypes
```
Comment (maarekua) [inline src/containers/constructor-features-container/constructor-features-container.js:2]: Then please do it.

File: src/containers/materials-with-color-container/materials-with-color-container.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
+/* eslint-disable react/prop-types */
+// TODO: Implement propTypes
```
Comment (maarekua) [inline src/containers/materials-with-color-container/materials-with-color-container.js:2]: Same here.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/containers/constructor-features-container/constructor-features-container.js (hunk 2 above)
Comment (Pipuss) [reply src/containers/constructor-features-container/constructor-features-container.js:2]: Implemented.

File: src/containers/materials-with-color-container/materials-with-color-container.js (hunk 3 above)
Comment (Pipuss) [reply src/containers/materials-with-color-container/materials-with-color-container.js:2]: Done it.

File: src/components/forms/material-form/material-form.js (hunk 1 above)
Comment (Pipuss) [reply src/components/forms/material-form/material-form.js:240]: Got it and fixed.

**Review by dmytruk-y — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

**Review by Tolik170 — DISMISSED** [review]

**Review by dmytruk-y — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**